schema/spacewalk/upgrade/susemanager-schema-4.1.0-to-susemanager-schema-4.1.1/350-delete-clone-info-in-content-projects.sql

```diff
@@ -0,0 +1 @@
+delete from rhnchannelcloned where id in (select c.id from rhnchannel c inner join susecontentenvironmenttarget et on c.id = et.channel_id);
```

Comment:
at least wrong directory. But better would be, if we would  not need it.

Reply:
Migration not needed (projects become consistent eventually).